onnxruntime/contrib_ops/cuda/bert/group_query_attention_helper.h

```diff
@@ -189,8 +189,12 @@ Status CheckInputs(const Tensor* query,
 
```

Comment:
You don't need to check key and value in the above code. They will be checked by ORT framework.